wsd/ClientSession.cpp

```diff
@@ -319,7 +319,7 @@ bool ClientSession::_handleInput(const char *buffer, int length)
     std::shared_ptr<DocumentBroker> docBroker = getDocumentBroker();
     if (!docBroker || docBroker->isMarkedToDestroy())
     {
-        LOG_ERR("No DocBroker found, or DocBroker marked to be destroyed. Terminating session " << getName());
+        LOG_FTL("No DocBroker found, or DocBroker marked to be destroyed. Terminating session " << getName());
```

Comment:
Nothing fatal. It's very common that a document being unloaded gets some activity that we reject (such as loading a new view). We continue working just fine, and this is reasonably common, but it's not FTL.